### ISSUE TITLE
Community indicator should be changed

### ISSUE THREAD

**Issue by FinecoFinit** [misskey-dev/misskey#7318]
In short, this black line is very distracting.
![image](https://user-images.githubusercontent.com/22540805/110239034-10823c00-7f6f-11eb-8cb2-768aa486bdff.png)

I think there should be only icon next to nickname

**Comment by cliqsocial** [comment]
Agree. We receive many complaints from our community and had to remove it. 

https://cliq.social/notes/8fpglr1h33
